dataactbroker/app.py

```diff
@@ -66,6 +68,16 @@ def before_request():
     def root():
         return "Broker is running"
 
+    @app.errorhandler(ResponseException)
+    def handle_response_exception(exception):
+        return JsonResponse.error(exception, exception.status)
+
+    @app.errorhandler(Exception)
+    def handle_exception(exception):
+        wrapped = ResponseException(str(exception), StatusCode.INTERNAL_ERROR,
+                                    type(exception))
+        return JsonResponse.error(wrapped, wrapped.status)
```

Comment:
I was wondering how we were going to handle the exceptions when we removed RouteUtils, great solution.